docs/PLANS.md

```
@@ -3,146 +3,184 @@
-world-agnostic.
-
-```rust
-#[fynix(interaction)]
-fn on_click_button(
-    e: &mut Button,
-    interaction: Click,
-    events: &mut Events,
-) {
-    (e.on_click)(events);
-}
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)